### PR TITLE
MLN-486: made language filter work on both primary_langiage and language colum…

### PR DESCRIPTION
**Pull request by daryanypl**
…ns, and added other languages field to teacher set detail page

### CODE REVIEW

File: app/models/teacher_set.rb [hunk 1]
```diff
@@ -160,10 +160,11 @@ def self.for_query(params)
       sets = sets.where("set_type = ?", params[:type])
     end
 
-    [:language, :availability].each do |prop|
-      if !params[prop].nil? && params[prop].size > 0
-        sets = sets.where("#{prop} IN (?)", params[prop])
-      end
+    if !params[:language].nil? && params[:language].size > 0
```
Comment (gonuguntla) [inline app/models/teacher_set.rb]: instead of !params[:language].nil?  we can use params[:language].present?

File: app/models/teacher_set.rb [hunk 2]
```diff
@@ -160,10 +160,11 @@ def self.for_query(params)
...
+      sets = sets.where("language IN (?) OR primary_language IN (?)", params[:language], params[:language])
+    end
+    if !params[:availability].nil? && params[:availability].size > 0
+      sets = sets.where("#{:availability} IN (?)", params[:availability])
```
Comment (gonuguntla) [inline app/models/teacher_set.rb]: #{} - this is not required for availability, this is column

File: app/models/teacher_set.rb [hunk 3]
```diff
@@ -219,6 +220,7 @@ def self.facets_for_query(qry)
         facets << facets_group
       end
 
+      #binding.pry
```
Comment (gonuguntla) [inline app/models/teacher_set.rb]: Please remove binding.pry

Comment (daryanypl) [reply app/models/teacher_set.rb]: done

File: app/models/teacher_set.rb (hunk 2 above)
Comment (daryanypl) [reply app/models/teacher_set.rb]: done

File: app/models/teacher_set.rb (hunk 1 above)
Comment (daryanypl) [reply app/models/teacher_set.rb]: done

**Pull request merged**